Client/Frontend/Widgets/NewTabButton.swift

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import SnapKit
+
+class NewTabButton: UIButton {
+    lazy var plusImage: UIImageView = {
+        let plusImage = UIImageView()
+        plusImage.image = UIImage(named: "menu-NewTab")?.tinted(withColor: .systemBlue)
+        return plusImage
+    }()
+    lazy var newTabCopy: UILabel = {
```

Comment:
nit: instead of `newTabCopy` could we change to `newTabTitle` or `openNewTabTitle` something more descriptive